### PR TITLE
Feat/app layout first pass

### PR DESCRIPTION
**Pull request by DanilFetter**

### CODE REVIEW

File: frontend/src/index.html [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body class="mat-typography">
```
Comment (kleputin) [inline frontend/src/index.html]: Is that class still needed?

File: frontend/src/styles.scss [hunk 2]
```diff
@@ -1,12 +1 @@
...
-
-.wrapper{
-  width: 100%;
-  background-color: #fff;
-  content: "";
-}
-
-html, body { height: 100%; }
-body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+html, body { height: 100vh; width: 100%; margin: 0; }
```
Comment (kleputin) [inline frontend/src/styles.scss]: Format it, please

File: frontend/src/index.html [hunk 3]
```diff
@@ -1,16 +1,28 @@
...
-  <link rel="preconnect" href="https://fonts.gstatic.com">
-  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-</head>
-<body class="mat-typography">
-  <app-root></app-root>
-</body>
+  <head>
+    <meta charset="utf-8" />
+    <title>Frontend</title>
```
Comment (kleputin) [inline frontend/src/index.html]: Change title to `TO-DO App`, please.

File: frontend/src/assets/delete-button.svg [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="18" viewBox="0 0 16 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (kleputin) [inline frontend/src/assets/delete-button.svg]: I think `trash.svg` will be more better name, didn't?

File: frontend/src/app/task-area/task-area.component.html [hunk 5]
```diff
@@ -1,22 +1,41 @@
...
+<div class="wrapper">
   <div class="working-area">
-    <form class="example-form">
-      <mat-form-field class="example-full-width" appearance="fill">
-        <mat-label>Print text of you task</mat-label>
-        <input matInput  />
-      </mat-form-field>
-      <button mat-button color="primary">Add new task</button>
-    </form>
+    <div class="side-bar">
```
Comment (kleputin) [inline frontend/src/app/task-area/task-area.component.html]: I think sidebar should be separate component.

File: frontend/src/app/task-area/task-area.component.html [hunk 6]
```diff
@@ -1,22 +1,41 @@
...
+      <button class="btn active-tab">All</button>
+      <button class="btn">Groceries</button>
+      <button class="btn">College</button>
+      <button class="btn">Payments</button>
+    </div>
+    <div class="task-area">
```
Comment (kleputin) [inline frontend/src/app/task-area/task-area.component.html]: Task area should be separate component too.

File: frontend/src/app/task-area/task-area.component.html [hunk 7]
```diff
@@ -1,22 +1,41 @@
...
+      <h1>All Tasks</h1>
+      <input
+        class="new-input"
+        type="text"
+        placeholder="Add a new task insdie ‘All’ category"
+      />
+      <div class="task-block">
```
Comment (kleputin) [inline frontend/src/app/task-area/task-area.component.html]: Task-block should be separate component or template (where you choose).

File: frontend/src/app/task-area/task-area.component.scss [hunk 8]
```diff
@@ -1,88 +1,261 @@
...
+  margin-right: 0.5em;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 50% 50%;
+}
+
+.custom-checkbox:checked+label::before {
+  border-color: #0b76ef;
+  background-color: #0b76ef;
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 8 8'%3e%3cpath fill='%23fff' d='M6.564.75l-3.59 3.612-1.538-1.55L0 4.26 2.974 7.25 8 2.193z'/%3e%3c/svg%3e");
```
Comment (kleputin) [inline frontend/src/app/task-area/task-area.component.scss]: so-bad-facepalm

**Review by kleputin — CHANGES_REQUESTED** [review]

File: frontend/src/app/task-area/task-area.component.html [hunk 9]
```diff
@@ -1,22 +1,41 @@
...
+            <input
+              type="checkbox"
+              [checked]="item.status"
+              class="custom-checkbox"
+              id={{item.id}}
+            />
+            <label for={{item.id}}></label>
+            <span class="task-text">{{item.text}}</span>
+            <div class="task-tab">{{item.tabGroupe}}</div>
+            <div class="gaper"></div>
```
Comment (kleputin) [inline frontend/src/app/task-area/task-area.component.html]: What is that?

File: frontend/src/app/task-area/task-area.component.scss [hunk 10]
```diff
@@ -1,88 +1,261 @@
...
+  align-items: flex-end;
   display: flex;
-  align-items: center;
+  justify-content: flex-end;
 }
 
-.icon{
-  width:25px;
-  height: 25px;
+@media (max-width:1058px) {
```
Comment (kleputin) [inline frontend/src/app/task-area/task-area.component.scss]: Let's focus on the desktop screens for now. I think you can remove all media queries.

**Review by kleputin — CHANGES_REQUESTED** [review]

File: frontend/src/app/task-area/task-area.component.html (hunk 9 above)
Comment (DanilFetter) [reply frontend/src/app/task-area/task-area.component.html]: fixed

File: frontend/src/app/task-area/task-area.component.scss (hunk 10 above)
Comment (DanilFetter) [reply frontend/src/app/task-area/task-area.component.scss]: deleted

File: frontend/src/app/task-area/task-area.component.scss (hunk 8 above)
Comment (DanilFetter) [reply frontend/src/app/task-area/task-area.component.scss]: stack overflow  moment

File: frontend/src/index.html (hunk 3 above)
Comment (DanilFetter) [reply frontend/src/index.html]: done

File: frontend/src/styles.scss (hunk 2 above)
Comment (DanilFetter) [reply frontend/src/styles.scss]: Done

File: frontend/src/index.html (hunk 1 above)
Comment (DanilFetter) [reply frontend/src/index.html]: removed

File: frontend/src/assets/delete-button.svg (hunk 4 above)
Comment (DanilFetter) [reply frontend/src/assets/delete-button.svg]: Already changed

File: frontend/src/index.html [hunk 11]
```diff
@@ -1,16 +1,27 @@
...
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+    <app-root></app-root>
```
Comment (kleputin) [inline frontend/src/index.html:26]: Where is open tag for body?) Is it compiles without errors?

**Review by kleputin — CHANGES_REQUESTED** [review]

File: frontend/src/app/task-area/task-area.component.html (hunk 5 above)
Comment (DanilFetter) [reply frontend/src/app/task-area/task-area.component.html]: done

File: frontend/src/app/task-area/task-area.component.html (hunk 6 above)
Comment (DanilFetter) [reply frontend/src/app/task-area/task-area.component.html]: done

File: frontend/src/app/task-area/task-area.component.html (hunk 7 above)
Comment (DanilFetter) [reply frontend/src/app/task-area/task-area.component.html]: done

File: frontend/src/index.html (hunk 11 above)
Comment (DanilFetter) [reply frontend/src/index.html:26]: fixed

**Pull request merged**